### PR TITLE
Adding_SAPIO registry.json

### PR DESCRIPTION
**Pull request by nikolas835**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -5096,5 +5096,18 @@
...
+ {
+	"ticker": "SAPIO",
+	"poolId": "c3cd2527a5f0b82a86ff69fdd5a3440dc51bcb957953ba19137738c0",
+	"social": {
+		"twitter": "nikos_SAPIO",
+		"facebook": "",
+		"instagram": "",
+		"youtube": "https://www.youtube.com/channel/UCcPH2RMsszRGJ2awvLdMKzQ"
+	},
+	"telegram": "Sapiopool",
```
Comment (Bez625) [inline registry.json:5109]: Please change this to be the telegram username of an individual and not a group or channel

File: registry.json [hunk 2]
```diff
@@ -5096,5 +5096,18 @@
    "telegram": "wafflepool",
    "operatorName": "",
    "github": ""
- }
+ },
+ {
+	"ticker": "SAPIO",
+	"poolId": "c3cd2527a5f0b82a86ff69fdd5a3440dc51bcb957953ba19137738c0",
+	"social": {
+		"twitter": "nikos_SAPIO",
```
Comment (Bez625) [inline registry.json:5104]: Looks like this account doesnt exist

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by nikolas835** [comment]
I've changed that now, had them reversed.
Thanks

**Comment by nikolas835** [comment]
Awesome, thanks!

**Pull request merged**